BanyanImages/res/pf_dispatch_table.toml

```diff
@@ -0,0 +1,17 @@
+# TODO: THIS NEEDS TO BE IMPLEMENTED
+
+
+[splits.ReadBlockPNG]
+expected = [ "key" ]
+partition_memory_usage = "part"
+scale_by = 2
```

Comment:
The `scale_by` here and below is correct :+1: . `scale_by` just refers to how much you need to scale the total amount of memory used to account for copying/extra allocation that might take place in the process of running the partitioning function.